apps/demo/src/hooks/useAccountsAccessToken.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
+
+declare global {
+  interface Window {
+    ocmConfig?: {
+      configData?: {
+        apiGateway?: string;
+      };
+    };
+  }
+}
+
+const getBaseUrl = (): string =>
+  window.ocmConfig?.configData?.apiGateway || 'https://api.openshift.com';
+
+export type AccountsAccessToken = {
+  auths: {
+    [domain: string]: {
+      auth: string;
+      email?: string;
+    };
+  };
+};
+
+export type OCMErrorResponse = {
+  kind: string;
+  id: string;
+  href: string;
+  code: string;
+  reason?: string;
+};
+
+export function objectIsAccessToken(
+  obj: AccountsAccessToken | OCMErrorResponse
+): obj is AccountsAccessToken {
+  return 'auths' in obj;
+}
+
+// Hook para obtener el token de acceso
+export function useAccountsAccessToken(): {
+  accessToken: string | undefined;
+} {
+  const [accessToken, setAccessToken] = React.useState<string | undefined>(undefined);
+  const { auth } = useChrome();
+
+  React.useEffect(() => {
+    async function getToken(): Promise<void> {
+      try {
+        const bearerToken = await auth.getToken(); // Get Bearer Token
+        if (bearerToken) {
+          const result = await fetchAccessToken(bearerToken);
```

Comment:
So we get token from the current browser and then send a request to `api/accounts_mgmt/v1/access_token` and we do it each time? 
So if will will stay on a page for longer then 15mintes(expiration of token) what happens?

Reply:
@machacekondra yes, we hace to pass the token to the request each time.
Good question, we have to test it in staging.